### DISCUSSION TITLE
Near-term plan; give me any sugession and opinion.

### DISCUSSION THREAD

**Discussion by vrtmrz** [vrtmrz/obsidian-livesync#112] [General]
I'm currently working on a few things.

- Log message improvement.
- Documentation improvement.
  - Or, Implement `Set up wizard` and decrease configuration items. 
- Split common logic into the library (Now 50%. We have to write test cases)

If you have any ideas. Please feel free to ask me.

And should I have an English Twitter account for notifications of releases?

**Comment by Vyeche** [comment]
@vrtmrz Perhaps a way to select multiple conflicts at once and resolve them.

![image](https://user-images.githubusercontent.com/972377/221994861-3fb2b9a8-c4c5-44ef-8484-5fd5dd69d0e5.png)

**Reply by vrtmrz** [reply]
I assume that it is not so frequently used (And, hope so), but might be useful for us now. It will be implemented under the hatch.

**Reply by Vyeche** [reply]
Yeah, not so frequent, I noticed that I need the new feature a lot more when there is a big release, and it sometimes breaks, and I have to redo the DB and creates modifications chaos. Resolving them one at a time takes too long, I have hundreds of files. 

Hopefully I can quickly select a bunch at a time and queue them for merge. Most of the time it's a date issue, but nothing to compare.

**Reply by vrtmrz** [reply]
The feature has been implemented as `Resolve all conflicted files`! To use your words, this acts as if all files have been queued.
To avoid unintended manipulation, at least one action is still needed for each conflicted document, but almost only we have to do is just keep tapping.

By the way, do you by any chance have the "Check conflicts only on opened files" (this is so weird English...) enabled?
If it has been disabled, all changes are tried to resolve automatically, otherwise, the automatic checking is proceed only for opened files. This configuration had been enabled by default for performance issues once, now disabled again.
Could you please check the configuration once?

**Reply by Vyeche** [reply]
It's a little faster, but as you said I need to keep clicking... 

How do I select a bunch of them at the same time? Like a combobox (see second screenshot)

![Screenshot_20230302-065846.png](https://user-images.githubusercontent.com/972377/222422630-fe4bb4a2-6d72-411f-af7a-08ede1097622.png)

![PXL_20230302_120439964.MP.jpg](https://user-images.githubusercontent.com/972377/222423916-ee398391-f5b8-4f2b-a2d2-a7ca6b8aaa04.jpg)

**Reply by Vyeche** [reply]
![PXL_20230302_120051230.jpg](https://user-images.githubusercontent.com/972377/222423016-b5e5d0be-9fa8-4946-afab-35acd5065cb5.jpg)

Is disabled

**Reply by hilsonp** [reply]
> The feature has been implemented as `Resolve all conflicted files`! To use your words, this acts as if all files have been queued. To avoid unintended manipulation, at least one action is still needed for each conflicted document, but almost only we have to do is just keep tapping.

Having to take an action on each file separately is cumbersome, really.
In the first place, looking at those tens of files out of sync is making me loose confidence but also, it asks me for each file to:
- press a hot key to get the 'pick a file to resolve conflict'
- select it
- say I want to keep A
- go to next one

Try to do this for a hundred files on an iphone, especially when the "keep A" is soo close to "keep B" that you never know which one you pushed with your fat finger :-/
